Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -9,7 +9,7 @@ class OWDistanceTransformation(widget.OWWidget):
     name = "Distance Transformation"
     description = "Transform distances according to selected criteria."
     icon = "icons/DistancesTransformation.svg"
-
+    
```

Comment:
No need for this whitespace either. You can set your editor to strip trailing (end-of-line) whitespace, at least on lines you touch.